Update printer.cfg

newphreak opened the pull request:
Update printer.cfg and make it default for LDO kits.
I used Voron 2.4's spider conf as template and tried to keep it as close to the values that were already in the conf, while making it a little easier to see what to change and so on.

newphreak committed:
Update printer.cfg

Update printer.cfg and make it default for LDO kits

newphreak committed:
Update printer.cfg

Clean up tabulars

nemgrea commented on Firmware/printer.cfg:
```diff
@@ -253,12 +282,12 @@ gcode:
-#   "AD595"
-#   "PT100 INA826"
-
+## 	Common Temperature Sensors
```
id reword this back to sensor types, klipper supports a certain list of sensor types "common" makes it sound like there may be others that would work here but i dont believe that is the case.

nemgrea commented on Firmware/printer.cfg:
```diff
@@ -83,108 +110,107 @@ uart_pin: PC11
-hold_current: 0.2	# for LDO 36STH17-1004AHG
+## For OMC (StepperOnline) 14HR07-1004VRN 1A
 #run_current: 0.5	# for OMC 14HR07-1004VRN rated at 1A
 #hold_current: 0.3	# for OMC 14HR07-1004VRN rated at 1A
```
chatted about this on discord

nemgrea commented on Firmware/printer.cfg:
```diff
@@ -83,108 +110,107 @@ uart_pin: PC11
 max_temp: 270
-min_extrude_temp: 170
-max_extrude_only_distance: 780.0
+min_extrude_temp: 190
```
i think some PLA's can run below this temp

nemgrea commented on Firmware/printer.cfg:
```diff
@@ -83,108 +110,107 @@ uart_pin: PC11
+#hold_current: 0.15
+## For LDO-42STH25-1004CL200E 1.0A
+run_current: 0.37
+hold_current: 0.35
```
chatted about this on discord

nemgrea commented on Firmware/printer.cfg:
```diff
@@ -34,22 +57,23 @@ uart_pin: PC11
 [stepper_y]
 step_pin: PB10
-dir_pin: PB2            # check https://docs.vorondesign.com/build/startup/#v0 if moving in the wrong direction
+dir_pin: PB2								# Add ! if moving opposite direction
```
this tip needs to get users to the homing direction graphic somehow because "wrong" direction for a single motor in corexy doesnt necessarily mean adding/removing the !

nemgrea commented on Firmware/printer.cfg:
```diff
@@ -9,23 +9,46 @@
 [stepper_x]
 step_pin: PB13
-dir_pin: PB12           # check https://docs.vorondesign.com/build/startup/#v0 if moving in the wrong direction
+dir_pin: PB12								# Add ! if moving opposite direction
```
this tip needs to get users to the homing direction graphic somehow because "wrong" direction for a single motor in corexy doesnt necessarily mean adding/removing the !

nemgrea reviewed (changes requested):
see notes in commit

newphreak committed:
Update printer.cfg

-Commented motor currents so user has to uncomment the correct one
-Changed min extrude temp back to 170
-Reworded and reworked thermistors

newphreak committed:
Update printer.cfg

Make X/Y A/B direction clearer

newphreak committed:
Update printer.cfg

fix braindead linking

hyperphonic0 commented on Firmware/printer.cfg:
```diff
@@ -34,22 +59,23 @@ uart_pin: PC11
 sense_resistor: 0.110
-stealthchop_threshold: 0
-
+stealthchop_threshold: 0                                            # Set to 9999 to turn off stealthchop
```
@newphreak this comment isn't right:

>  "# Set to 9999 to turn off Stealthchop" 

Stealthchop is **OFF** by default - the default value for `stealthchop_threshold` is 0.

Specifying a non-zero value `stealthchop_threshold` turns Stealthchop **ON** for velocities **LOWER** than it.

newphreak replied:
eek, that is totally correct sir.

hyperphonic0 commented on Firmware/printer.cfg:
```diff
@@ -83,108 +112,120 @@ uart_pin: PC11
+## For LDO-42STH25-1004CL200E 1.0A
+#run_current: 0.37
 #hold_current: 0.35
-run_current: 0.2         # For V0.1 spec NEMA17 OMC 17LS13-0404E-200G 0.4A 
```
@newphreak You now have NO `run_current` specified at all for the Z stepper!

newphreak replied:
This was intentional to make the user choose the correct one and not trust anything blindly

newphreak replied:
it is also stated clearly in the top what you should look at. ideas to make it clearer are more then welcome

hyperphonic0 commented on Firmware/printer.cfg:
```diff
@@ -58,23 +84,26 @@ uart_pin: PC11
 hold_current: 0.25
 sense_resistor: 0.110
-stealthchop_threshold: 0
+stealthchop_threshold: 0                                            # Set to 9999 to turn on stealthchop
```
Same applies here.

> @newphreak this comment isn't right:
> 
> "# Set to 9999 to turn off Stealthchop"
> Stealthchop is OFF by default - the default value for stealthchop_threshold is 0.
> 
> Specifying a non-zero value stealthchop_threshold turns Stealthchop ON for velocities LOWER than it.

hyperphonic0 replied:
@newphreak this comment isn't quite right. According to the Klipper docs, if specified that value should be `stealthchop_threshold: 999999` or `stealthchop_threshold: 0`.

hyperphonic0 commented on Firmware/printer.cfg:
```diff
@@ -83,108 +112,120 @@ uart_pin: PC11
-hold_current: 0.15
 sense_resistor: 0.110
-stealthchop_threshold: 0
+stealthchop_threshold: 0                                            # Set to 9999 to turn on stealthchop
```
as above:

>  @newphreak this comment isn't quite right. According to the Klipper docs, if specified that value should be `stealthchop_threshold: 999999` or `stealthchop_threshold: 0`.

newphreak committed:
Update printer.cfg

reword stealthchop/spreadcycle

newphreak committed:
Update printer.cfg

Minor description updates

newphreak committed:
Update printer.cfg

Consistency
